File: backend/components/metiers/administration_publique/utils/i18n/helpers/i18n_helper.test.py
# i18n_helper.test.py
# Tests unitaires Python pour i18n_helper
from .i18n_helper import humanize_key

def test_humanize_key():
    assert humanize_key('HELLO_WORLD') == 'hello world'

def test_humanize_key_empty():
    assert humanize_key('') == ''

File: backend/components/metiers/administration_publique/utils/i18n/helpers/i18n_helper.test.js
// i18n_helper.test.js
// Tests unitaires JS pour i18n_helper
const { humanizeKey } = require('./i18n_helper');

describe('humanizeKey', () => {
  it('transforme une clé i18n en format lisible', () => {
    expect(humanizeKey('HELLO_WORLD')).toBe('Hello world');
  });
  it('gère le cas vide', () => {
    expect(humanizeKey('')).toBe('');
  });
});
